docs/pvc_scenario.md

```diff
@@ -0,0 +1,21 @@
+### PVC scenario
+Scenario to fill up a given PersistenVolumeClaim by creating a temp file of a given size from a pod that has the volume mounted in the given path. The purpose of this scenario is to fill up a volume to understand faults cause by the application using this volume. 
+
+##### Sample scenario config
+```
+pvc_scenario:                        # Scenario to stop all the nodes for specified duration and restart the nodes
```

Comment:
Comment on this line does not match the scenario

Reply:
done